int8 PUSH = 1
int8 POP = 2
int8 mode
QueueElement item